README.md

```diff
@@ -145,3 +145,13 @@ You can access emails in MailHog's Web UI at [http://localhost:8075](http://loca
 
 - [Documentation](https://docs.camunda.org/)
 - [GitHub](https://github.com/camunda/camunda-bpm-platform)
+
+# Troubleshooting
+## Running on arm64 based hardware
+When using arm64-based hardware like a M1 or M2 Mac the Keycloak container might not start because Bitnami only
+provides amd64-based images. You can build and tag an arm-based image locally using following command. After building
```

Comment:
❓ wdyt about linking the bitnami feature request here for easily navigating to it in future, like 
```suggestion
provides amd64-based images. Until bitnami adds [support for linux/arm64 images](https://github.com/bitnami/charts/issues/7305), you can build and tag an arm-based image locally using following command. After building
```

Reply:
Good suggestion, added 👍